### PR TITLE
Support UI feature set

### PR DESCRIPTION
**Pull request by colin014**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| API breaks?     | no
| Deprecations?   | no
| License         | Apache 2.0

### What's in this PR?
Support UI feature set via UI proxy

### Why?
The new UI version(s) use the new feature set (by default everything is turned off). To use UI in local turn on these features.

- [x] Implementation tested (with at least one cloud provider)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -35,7 +35,7 @@ GOLANG_VERSION = 1.11.5
 GOFILES_NOVENDOR = $(shell find . -type f -name '*.go' -not -path "./vendor/*" -not -path "./client/*")
 
 .PHONY: up
-up: config/dex.yml start config/config.toml ## Set up the development environment
+up: config/dex.yml config/ui/feature-set start config/config.toml ## Set up the development environment
```
Comment (sagikazarmark) [inline Makefile]:
```suggestion
up: config/dex.yml config/ui/feature-set.json start config/config.toml ## Set up the development environment
```

File: Makefile [hunk 2]
```diff
@@ -66,6 +66,9 @@ stop: ## Stop docker development environment
 config/config.toml:
 	cp config/config.toml.dist config/config.toml
 
+config/ui/feature-set:
```
Comment (sagikazarmark) [inline Makefile]:
```suggestion
config/ui/feature-set.json:
```

File: Makefile [hunk 3]
```diff
@@ -66,6 +66,9 @@ stop: ## Stop docker development environment
...
+	cp config/ui/feature-set.json.dist config/ui/feature-set.json
```
Comment (sagikazarmark) [inline Makefile]:
```suggestion
	cp -b config/ui/feature-set.json.dist config/ui/feature-set.json
```

File: etc/docker/uiproxy/nginx.conf [hunk 4]
```diff
@@ -54,5 +54,10 @@ http {
             proxy_set_header Host            $http_host;
             proxy_set_header X-Forwarded-For $remote_addr;
         }
+
+        location /assets {
+             root /usr/share/nginx/html/;
+        }
+
```
Comment (sagikazarmark) [inline etc/docker/uiproxy/nginx.conf]:
```suggestion
```

File: etc/docker/uiproxy/nginx.conf [hunk 5]
```diff
@@ -54,5 +54,10 @@ http {
             proxy_set_header Host            $http_host;
             proxy_set_header X-Forwarded-For $remote_addr;
         }
+
+        location /assets {
+             root /usr/share/nginx/html/;
```
Comment (sagikazarmark) [inline etc/docker/uiproxy/nginx.conf]:
```suggestion
            root /usr/share/nginx/html/;
```

**Comment by colin014** [comment]
thx @sagikazarmark, fixed

**Pull request merged**